src/structures/typedefs.h

```diff
@@ -94,6 +95,12 @@ struct HeuristicParameters {
                                 float regret_coeff)
     : heuristic(heuristic), init(init), regret_coeff(regret_coeff) {
   }
+
+  // Only makes sense for user-defined initial routes.
+  constexpr HeuristicParameters(HEURISTIC heuristic)
+    : heuristic(heuristic), init(INIT::NONE), regret_coeff(0) {
+    assert(heuristic = HEURISTIC::INIT_ROUTES);
```

Comment:
I think this is intended to be a comparison.
```suggestion
    assert(heuristic == HEURISTIC::INIT_ROUTES);
```

Reply:
Of course, good catch! By coincidence and due to the current usage of the ctor, this has no effect. But it may hide an unintended use of the ctor at some point in the future, which was the reason for having an assert in the first place...

@M393 do you fancy submitting this change as a simple PR?

Reply:
I opened #609 with this fix.